dev/preview/infrastructure/harvester/cloudinit.yaml

```diff
@@ -18,11 +18,13 @@ write_files:
       systemctl disable google-osconfig-agent &
       systemctl disable google-oslogin-cache.timer &
       systemctl disable google-shutdown-scripts &
+      systemctl disable load-workspace-full.service &
```

Comment:
As this isn't required for the GCE-based preview environments, let's not change this yet. My concern is that it could negatively affect integration tests or use-cases we are not aware of, so having it in a separate PR seems best to me.